open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -119,19 +119,32 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        return df[df[column_name] == column_value]
</pre><p>Please just revert the changes in this cell. You don't need to add such more checks. They are not necessary. Also, you don't need to check <code>if column_value not in df[column_name].values:</code></p><p> </p><p>But all your new added test cases are very good. You can keep them, but you may need to fix the assert a little bit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done</p>